addons/account/models/sequence_mixin.py

```diff
@@ -126,6 +126,13 @@ def _compute_split_sequence(self):
             record.sequence_number = int(matching.group(1) or 0)
 
     @api.model
+    @ormcache(
+        "name",
+        "self._sequence_fixed_regex",
+        "self._sequence_monthly_regex",
+        "self._sequence_year_range_regex",
+        "self._sequence_yearly_regex",
+    )
```

Comment:
This is complete nonsense.
Do you know what `ormcache` does?
* `self._sequence_` are not likely to change. Explicitly invalidating the cache when those values change would make more sense that losing all that memory to store the (useless) keys.
* `name` has literally an infinite possible amount of values, and they are very unlikely to be reused again in future transactions.
* the function also depends on `where_string`, which is dynamically computed based on various fields on `self`, which is not that easy to use in the key
* this function doesn't even appear on an Profiler when creating and posting moves... (or barely)

Reply:
Haha ok, let's forget about this part. I guess I was optimizing here and there and this just remained as it didn't give problems. But yes, the main performance problem is in the query.